docs/docs/components/page-shell.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+// docs-page-shell
+import ReactPageShell from '../../vendor/docs-page-shell/react-page-shell.js';
+// dr-ui components
+import PageLayout from '@mapbox/dr-ui/page-layout';
+import { buildMeta, findParentPath } from '@mapbox/dr-ui/page-layout/utils';
+// site variables
+import constants from '../constants';
+// batfish modules
+import { withLocation } from '@mapbox/batfish/modules/with-location';
+// dataSelectors
+import navigation from '@mapbox/batfish/data/navigation';
+import filters from '@mapbox/batfish/data/filters';
+
+import { styleSpecNavigation } from '../data/style-spec-navigation';
+
+import AppropriateImage from './appropriate-image';
+import Browser from '@mapbox/dr-ui/browser';
+import classnames from 'classnames';
+import { version as styleSpecVersion } from '../../../package.json';
+
+const redirectStyleSpec = require('../util/style-spec-redirect');
+
+class PageShell extends React.Component {
```

Comment:
I'm not sure this should be here, this looks like the definition of the main react page in the docs.
If we want to make it modular, we can have a look at the direction of micro front ends, or simply move to a page containing the style-spec definitions without all the other parts of the docs.
Bottom line, this is probably duplicated between here and the other docs repo and there's a need to better restructure this...